index.Rmd

```diff
@@ -8,7 +8,7 @@ site: sandpaper::sandpaper_site
 
 <div style="text-align: center; margin-top: 30px; margin-bottom: 30px;">
 
-![](./fig/DC-logo-vision.png)
+![](episodes/fig/DC-logo-vision.png){alt=""}
```

Comment:
Note: I also updated the link to point to the correct place and added `alt=""` to indicate that this is a decorative image.